packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -72,6 +72,8 @@ const ActionList = ({ children }: ActionListProps): JSX.Element => {
     value: string;
   }[] = [];
 
+  console.count('ActionList');
```

Comment:
Stray log. Out of curiosity, did memoizing the context value have significant effect on re-renders?

Reply:
nope didn't see any major difference. (The stray log I ended up removing while pulling this branch in #966 )